alternator/executor.cc

```diff
@@ -5860,32 +5871,43 @@ static lw_shared_ptr<keyspace_metadata> create_keyspace_metadata(std::string_vie
     }
     auto opts = get_network_topology_options(sp, gossiper, rf);
 
-    // Even if the "tablets" experimental feature is available, we currently
-    // do not enable tablets by default on Alternator tables because LWT is
-    // not yet fully supported with tablets.
-    // The user can override the choice of whether or not to use tablets at
-    // table-creation time by supplying the following tag with a numeric value
-    // (setting the value to 0 means enabling tablets with automatic selection
-    // of the best number of tablets).
+    // Whether to use tablets for the table (actually for the keyspace
+    // of the table) is determined by the parameter `tablets_mode` (which is
+    // determined by `tablets_mode_for_new_keyspaces` config flag), as well as
+    // the presence and the value of a per-table tag below.
```

Comment:
Instead of the vague "below" I would write "WRITE_ISOLATION_TAG_KEY" (or `system:initial_tablets`).